### ISSUE TITLE
zfs-0.8.4: undefined symobol aok in libnvpair.so

### ISSUE THREAD

**Issue by RiderALT** [openzfs/zfs#10353]
Hi. 
found during  zfs 0.8.4 build for ALT Linux (ALT uses strong symbols checking in the build farm):
```
$ ldd -r lib/libnvpair/.libs/libnvpair.so
        linux-vdso.so.1 (0x00007fffc076f000)
        libc.so.6 => /lib64/libc.so.6 (0x00007ff6b9a4f000)
        /lib64/ld-linux-x86-64.so.2 (0x00007ff6b9c30000)
undefined symbol: aok   (lib/libnvpair/.libs/libnvpair.so)
```
adding int oak = 0; to lib/libnvpair/libnvpair.c solves this problem.

**Comment by behlendorf** [comment]
Related to #9752 .  That seems to be to simplest fix, would you mind opening a PR?

**Comment by meilihao** [comment]
me too with zfs-0.8.6 + ubuntu 16.04.6 + aarch64.

**Comment by behlendorf** [comment]
Closing.  This was resolved in the 2.0 release.  If possible I'd suggest moving to the new release.
